sanowret-crystal.lic

```diff
@@ -0,0 +1,39 @@
+=begin
+	Runs in the background, gazing or exhaling your sanowret crystal. Checks for concentration and current mindstate.
+
+	author: NBSL
+	game: dr
+  version: 0.1 (2017-06-12)
+
+  changelog:
+    0.1 (2017-06-12)
+     * Initial Release
+=end
+
+no_kill_all
+no_pause_all
+
+custom_require.call(%w[common drinfomon])
+invalid_room = nil
+
+while true do
+  line = script.gets?
+  response = ""
```

Comment:
We could add a line `next if invalid_room == Room.current.id` right here and simplify our later if statement.